### DISCUSSION TITLE
Anyone have an idea on the time/amount of HCs he killed?

### DISCUSSION THREAD

**Discussion by TelumehtarCC** [chromiecraft/chromiecraft#7289] [General]
![image](https://github.com/user-attachments/assets/63a73c34-c070-4231-8544-56983af1fd02)

**Comment by BeefSteaker** [comment]
One one was reported, but anyone who has lost a character to this bug can make a ticket and we'll restore them

**Comment by vvranceanu** [comment]
Going to leave this here

![image](https://github.com/user-attachments/assets/d73d4589-e6ec-4674-b98f-7903145e9a72)

**Comment by Nyeriah** [comment]
If there are more characters affected, please open tickets so we can address it.
Player reports are done through tickets as we don't condone public call-outs.

Closing this as the issue seems to have been dealt with.
